test/fixtures/custom-db.js

```diff
@@ -14,7 +14,7 @@ exports.generate = co.wrap(function * (tableName, tableSchema, bookshelfConfig)
   return bookshelf
 })
 
-exports.testTable = co.wrap(function * (bookshelfConfig) {
+exports.altFieldTable = co.wrap(function * (bookshelfConfig) {
```

Comment:
The rename is to indicate the specific modification applied to this table. Since I've added the sentinel-ized table below, I felt that this instance could use a name that indicated the change (field-name modification) rather than the generic "test", when it's actually only used for one specific test.